variables.tf

```diff
@@ -116,6 +116,65 @@ variable "tags" {
   default     = {}
 }
 
+variable "vpc_block_public_access_options" {
+  description = <<EOF
```

Comment:
Omg, please keep the descriptions as similar to the rest of the module's variables as possible. We normally show available options in the examples or README, but not in the description.

Update the list of the examples in the README.md

Reply:
I updated `README.md` and variables descriptions.